src/routes/businesses.ts

```diff
@@ -0,0 +1,289 @@
+    };
+
+    createdBusiness = await saveBusiness(business).then(() => {
+      res.status(200).send({
+        id: businessId,
+        ...req.body,
+      });
+    });
+
+    await storeBusinessInSortedSet(business);
+
+    if (business.account?.id) {
+      await saveAccountToBusinessId(business.account, businessId);
+    }
+
+    if (req.headers.origin) {
+      await setBusinessOrigin(businessId, req.headers.origin);
+    }
+  });
+
+  return createdBusiness;
+};
+
+export const deleteBusiness = async (req, res) => {
+  const { id: businessId } = req.params;
+  try {
+    await removeBusiness(businessId);
+    res.sendStatus(HttpStatusCodes.NO_CONTENT);
+  } catch (err) {
+    return res.status(500).send({
+      errors: [
+        {
+          id: ERROR_NOT_FOUND_ID,
+          status: 500,
+        },
+      ],
+    });
+  }
+};
+
+export const showBusiness = async (req, res) => {
+  const { business_id: businessId } = req.params;
+  try {
+    const business = await getBusiness(businessId);
+
+    return res.status(200).send(business);
+  } catch (err) {
+    if (err.message === "did not find business") {
+      const resp = {
+        errors: [
+          {
+            id: ERROR_NOT_FOUND_ID,
+            status: 404,
+            code: "model_not_found",
+            title: "Model Not Found",
+            detail: `Couldn't find 'Solaris::Business' for id '${businessId}'.`,
+          },
+        ],
+      };
+
+      return res.status(404).send(resp);
+    }
+
+    return res.status(500).send({
+      errors: [
+        {
+          id: ERROR_NOT_FOUND_ID,
+          status: 500,
+        },
+      ],
+    });
+  }
+};
+
+export const showBusinesses = async (req, res) => {
+  const { page: { size = 10, number = 1 } = {} } = req.query;
+
+  const businesses = ((await findBusinesses()) || []).slice(
+    (number - 1) * size,
+    size * number
+  );
+
+  return res.status(200).send(businesses);
+};
+
+export const BUSINESS_UPDATE = "Patch/Business/business_id";
+
+/**
+ * Checks if the model has setted previously a value given in the input.
+ * This is useful to check is a Solaris entity may be updated or not checking the full entity
+ * or the desired part of the entity.
+ * i.e isChangeRequestRequired(mydata, business) or isChangeRequestRequired(mydata, business.address)
+ * @param {$Request} req
+ * @param {$Response} res
+ */
+const isChangeRequestRequired = (input, model) => {
+  let flag = false;
+
+  if (input && model) {
+    Object.keys(input).forEach((key) => {
+      if (typeof input[key] === "object" && model[key]) {
+        flag = flag || isChangeRequestRequired(input[key], model[key]);
+      } else if (model[key]) {
+        flag = true;
+      }
+    });
+  }
+
+  return flag;
+};
+
+export const updateBusiness = async (req, res) => {
+  // Solaris responds with a 403 when the parameter being updated is "empty",
+  // i.e., `null`. This is not yet implemented here.
+  const fields = [
+    "name",
+    "title",
```

Comment:
could you compare this list with the one from solaris docs? there is no 'title' and 'origin'